pom.xml

```diff
@@ -12,6 +12,7 @@
 
     <properties>
         <target.jdk>17</target.jdk>
+        <jib.base.image>eclipse-temurin:20-jre-jammy</jib.base.image>
```

Comment:
21 jre should be more stable?

Reply:
I think the base image in the poms with jib it's this one. Would be probably better to upgrade it (well if Tung wants it's cool) but it isn't the goal here :)